src/decide.ts

```diff
@@ -102,6 +104,48 @@ export function compareFileTreesIgnoringNames(
 	);
 }
 
+export function compareFileTreesPartialIgnoringNames(
+	candidate: Metafile,
+	searchee: Searchee
+): boolean {
+	const { fuzzySizeThreshold } = getRuntimeConfig();
+	let matchedSizes = 0;
+	for (const candidateFile of candidate.files) {
+		let matchedSearcheeFiles = searchee.files.filter(
+			(searcheeFile) => searcheeFile.length === candidateFile.length
+		);
```

Comment:
it looks like we only use `!!matchedsearcheefiles.length` so this can be changed to `some` which will short-circuit after the first one it finds

```suggestion
		let matchedSearcheeFiles = searchee.files.some(
			(searcheeFile) => searcheeFile.length === candidateFile.length
		);
```